services/web/server/src/simcore_service_webserver/projects/projects_nodes_handlers.py

```diff
@@ -191,6 +212,72 @@ async def retrieve_node(request: web.Request) -> web.Response:
     )
 
 
+@routes.post(f"/{VTAG}/projects/{{project_id}}/nodes/{{node_id}}:start")
+@login_required
+@permission_required("project.update")
+async def start_node(request: web.Request) -> web.Response:
+    """Has only effect on nodes associated to dynamic services"""
+    req_ctx = RequestContext.parse_obj(request)
+    path_params = parse_request_path_parameters_as(_NodePathParams, request)
+    try:
+
+        await projects_api.start_project_node(
+            request, req_ctx.user_id, path_params.project_id, path_params.node_id
+        )
+
+        return web.HTTPNoContent(content_type=MIMETYPE_APPLICATION_JSON)
+    except ProjectStartsTooManyDynamicNodes as exc:
+        raise web.HTTPConflict(reason=f"{exc}") from exc
+    except ProjectNotFoundError as exc:
+        raise web.HTTPNotFound(
+            reason=f"Project {path_params.project_id} not found"
+        ) from exc
+    except NodeNotFoundError as exc:
+        raise web.HTTPNotFound(
+            reason=f"Node {path_params.node_id} not found in project"
+        ) from exc
+
+
+async def _stop_dynamic_service_with_progress(
+    _task_progress: TaskProgress, path_params: _NodePathParams, *args, **kwargs
+):
+    try:
+        await director_v2_api.stop_dynamic_service(*args, **kwargs)
+        return web.HTTPNoContent(content_type=MIMETYPE_APPLICATION_JSON)
+    except ProjectNotFoundError as exc:
```

Comment:
We should have an error handler that maps any ``projects_exceptions.Project*Error/Node*Error`` into http errors and apply this in all project handlers as decorator